burn-tensor/src/tensor/ops/modules/base.rs

```diff
@@ -299,6 +316,52 @@ pub trait ModuleOps<B: Backend> {
     ) -> B::TensorPrimitive<3> {
         pool::adaptive_avg_pool1d_backward_from_2d::<B>(x, grad)
     }
+    /// One dimensional max pooling.
+    ///
+    /// # Shapes
+    ///
+    /// x: [batch_size, channels, length],
+    fn max_pool1d(
+        x: B::TensorPrimitive<3>,
+        kernel_size: usize,
+        stride: usize,
+        padding: usize,
+    ) -> B::TensorPrimitive<3> {
+        pool::max_pool1d_from_max_pool2d::<B>(x, kernel_size, stride, padding)
+    }
+
+    /// One dimensional max pooling with indices.
+    ///
+    /// # Shapes
+    ///
+    /// x: [batch_size, channels, height, width],
+    fn max_pool1d_with_indices(
+        x: B::TensorPrimitive<3>,
+        kernel_size: usize,
+        stride: usize,
+        padding: usize,
+    ) -> MaxPool1dWithIndices<B> {
+        pool::max_pool1d_with_indices_from_max_pool2d::<B>(x, kernel_size, stride, padding)
+    }
+    /// Backward pass for the [max pooling 1d](ModuleOps::max_pool1d_with_indices) operation.
+    fn max_pool1d_with_indices_backward(
+        x: B::TensorPrimitive<3>,
+        kernel_size: usize,
+        stride: usize,
+        padding: usize,
+        output_grad: B::TensorPrimitive<3>,
+        indices: B::IntTensorPrimitive<3>,
+    ) -> MaxPool1dBackward<B> {
+        pool::max_pool1d_with_indices_backward_from_max_pool2d::<B>(
```

Comment:
I think just finishing the functions name with `from_2d` is enough.

`max_pool1d_from_max_pool2d` -> `max_pool1d_from_2d`
`max_pool1d_with_indices_from_max_pool2d` -> `max_pool1d_with_indices_from_2d`
`max_pool1d_with_indices_backward_from_max_pool2d` -> `max_pool1d_with_indices_backward_from_2d`